project-reports/2024/2024-Q3-Hyperledger-FireFly.md

```diff
@@ -0,0 +1,117 @@
+
+### Contributor activity updates
+
+The star activity for the FireFly repository has continued to grow from 481 end of last quarter to 496 end of this quarter. Discord activity has been very high this quarter with ~50 DAU.
+
+![alt_text](images/HLFF_2024_Q3_stars.png "image_tooltip")
+
+Contributors are active in the project and since June, there were 39 contributors who performed at least one commit.
+There are currently over 775k lines of code for Hyperledger FireFly across 20 repositories, with a total of 10.861k commits to date.
+
+# Questions/Issues for the TOC
+
+None
+
+# Releases
+
+This quarter, FireFly v1.3.1 was released with FireFly v1.3.2 upcoming next quarter. A high level overview of what’s new is below:
+
+## [FireFly v1.3.2 pre-release](https://github.com/hyperledger/firefly/releases/tag/v1.3.2-rc.1)
+
+FireFly v1.3.2 pre-release features includes the ability to install the FireFly CLI through Brew, a package manager for MacOS, bug fixes, documentation updates, and FireFly dependency updates. Look for a full update next quarter.
+
+## [FireFly v1.3.1](https://github.com/hyperledger/firefly/releases/tag/v1.3.1)
+
+FireFly v1.3 new features include:
+- Multi-filters per listeners: Added the ability to listen to multiple types of events on the same contract listener.
+- New Listener Type: Block listeners are added to stream receipts of block confirmations in the order that they are confirmed on the blockchain.
+- Improved Documentation: Missing documentation and README files have been added, including updates to doc links.
+- Multiparty Status API: A new API was added for managing multiparty transaction status.
+- Ethereum Blank Signing Key Support: Queries on Ethereum can now allow blank signing keys.
+- Enhanced Listener Management: When re-creating listeners in EVMConnect, EthConnect, or FabConnect, the last event block is now used to prevent duplicate handling.
+- Namespace Management: Fixes related to preventing multiple blockchain plugins from starting on namespace retry.
+- Websocket Enhancements: Websockets are now available as an option to communicate with EVM nodes in addition to RPC calls.
+- Performance: Event indexing optimized and bugs fixed to ensure that events and transactions are received and in the correct order.
+- Receipt Confirmation: Improved the receipt confirmations logic from the chain for EVM.
+
+For a full list of changes and updates, please visit [the FireFly Github](https://github.com/hyperledger/firefly/releases/tag/v1.3.1).
+
+# Overall Activity in the Past Quarter
+
+The major highlights have been detailed at the beginning of the report. In summary, there has been a lot of activity in the project. New community members continue to discover the project and make contributions. This quarter we have continued to see contribution and activities from outside the set of the original maintainers. A new Cardano Connector has been discussed to be in development by an external party.
```

Comment:
Would this external party join the maintainers, increase diversity with the Cardano Connector contribution?